lib/hsa/hsaBeamformHFBZeroData.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef HSA_HFB_ZERO_DATA_H
+#define HSA_HFB_ZERO_DATA_H
+
+#include "hsaSubframeCommand.hpp"
+
+class hsaBeamformHFBZeroData : public hsaSubframeCommand {
```

Comment:
This file is missing comments.  Although I'm wondering is it possible to avoid having to zero the output by adjusting the kernel in some way?